play/dbt-materialize/README.md

```diff
@@ -0,0 +1,62 @@
+## dbt-materialize
```

Comment:
if this is off-base, ok to ignore, but I would expect some kind of snippet of how `dbt` users can use Materialize somewhere––like maybe a code block showing what the correct config file looks like for a working instance? Maybe it's too trivial to include, idk

Reply:
Not off base at all! If I understand what you're suggesting, this actually exists in `sample_profiles.yml`. dbt users will know to look there, but I will add a link.